### ISSUE TITLE
build: go-libedit cannot find -lncurses

### ISSUE THREAD

**Issue by mjibson** [cockroachdb/cockroach#24625]
Is this a question, feature request, or bug report?

**QUESTION**

Have you checked our documentation at https://cockroachlabs.com/docs/stable/? If you could not find an answer there, please consider asking your question in our community forum at https://forum.cockroachlabs.com/, as it would benefit other members of our community.

Prefer live chat? Message our engineers on our Gitter channel at https://gitter.im/cockroachdb/cockroach.

**FEATURE REQUEST**

1. Does an issue already exist addressing this request? If yes, please add a :+1: reaction to the existing issue. If not, move on to step 2.

2. Please describe the feature you are requesting, as well as your proposed use case for this feature.

3. Indicate the importance of this issue to you (blocker, must-have, should-have, nice-to-have). Are you currently using any workarounds to address this issue?

**BUG REPORT**

1. Please supply the header (i.e. the first few lines) of your most recent
   log file **for each node in your cluster**. On most unix-based systems
   running with defaults, this boils down to the output of

     grep -F '[config]' cockroach-data/logs/cockroach.log

   When log files are not available, supply the output of `cockroach version`
   and all flags/environment variables passed to `cockroach start` instead.

2. Please describe the issue you observed:

- What did you do?

- What did you expect to see?

- What did you see instead?

**Comment by mjibson** [comment]
Happening on the 2.0 branch also.

**Comment by mjibson** [comment]
Fixed with a `apt install libncurses5-dev` on my system. But this is getting real old.

**Comment by knz** [comment]
Dear Matt,
I am sorry you just got caught in the fallout of #24513. 
I was careful to update CONTRIBUTING.md to report the changed external dependency, but I made the mistake to not announce it on the forum and the mailing list. 
My apologies for the insufficient communication.

Can you make a suggestion about how you would prefer to see this issue resolved?

**Comment by benesch** [comment]
Yeah, the Go readline package caused more problems than it solved. :(
